data.sql

```diff
@@ -1,5 +1,6 @@
 /* Populate database with sample data. */
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO animals VALUES ('0', 'Agumon', '2020-02-03', '0', 'true', '10.23');
+INSERT INTO animals VALUES ('1', 'Gabumon', '2018-11-15', '2', 'true', '8');
+INSERT INTO animals VALUES ('2', 'Pikachu', '2021-01-07', '1', 'false', '15.04');
+INSERT INTO animals VALUES ('3', 'Devimon', '2017-05-12', '5', 'true', '11');
```

Comment:
- [ ] As mentioned by the previous review and for the project requirements, kindly ensure that you correct data and correctly setup as right now you miss column names for each value. Below is an example of how it should look:
```sql
INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
VALUES ('Agumon', '2020/2/3', 0, true, '10.23');
```

Reply:
Hey @tresorsawasawa , I don't think you have looked carefully, the previous CR didn't mention changing these codes like that. even though I still have changed the code as you mentioned. although other students' projects were approved just like I submitted them first. It's frustrating. Please review.